src/sentry/api/event_search.py

```diff
@@ -2229,6 +2229,14 @@ def result_type_fn(function_arguments, parameter_values):
         ),
     ]
 }
+# In Performance TPM is used as an alias to EPM
+FUNCTION_ALIASES = {
+    "tpm": "epm",
+    "tps": "eps",
+}
+for alias, name in FUNCTION_ALIASES.items():
+    FUNCTIONS[alias] = deepcopy(FUNCTIONS[name])
```

Comment:
Instead of a deepcopy here, can this be a method on the `Function` class like `alias_as` or something?

Reply:
:+1: updated in 949ec86